app/create_file.py

```
@@ -1 +1,68 @@
-# write your code here
+import datetime
+import os
+import sys
+
+
+def get_params(args: list) -> dict:
+    """
+        Takes a list of arguments and returns
+        a dictionary of parameters with corresponding values.
+
+        Args:
+            args (list):
+                List of command-line arguments.
+
+        Returns:
+            dict:
+                A dictionary of parameters where
+                keys are flags (e.g., "-d", "-f") and
+                values are corresponding parameter values.
+    """
+
+    params = {"-d": "", "-f": ""}  # Initial parameter values
+    flag = None
+
+    for arg in args:
+        if arg in params:
+            flag = arg
+        elif flag in params:
+            params[flag] = os.path.join(params[flag], arg)
+
+    return params
+
+
+def create_file() -> None:
+    """
+        Creates a file based on the specified parameters
+        and adds content to it.
+
+        This function reads parameters from command-line arguments,
+        creates a directory if it doesn't exist, constructs the file path,
+        and appends content lines with line numbers and a timestamp.
+
+        The content lines are entered by the user
+        until the input "stop" is given.
+    """
+
+    params = get_params(sys.argv)
+
+    if not os.path.exists(params["-d"]):
+        os.makedirs(params["-d"])
```

os.makedirs(params["-d"], exist_ok=True)
This will ignore the exception when the directory exists